byway/src/main/webapp/index.js

```diff
@@ -0,0 +1,22 @@
+window.onload = function(){ 
+  fetch("/login").then(response => response.json()).then((login) =>{
+  if (login.isLoggedIn == true) {
```

Comment:
`isLoggedIn` is already a boolean. There's no need to add truth comparisons to booleans.

Reply:
fixed